recipe/meta.yaml

```diff
@@ -18,20 +18,22 @@ source:
     # 2022/07 hmaarrfk
     # We've patch out the bug in GCC that makes it incompatible
     # https://github.com/conda-forge/ctng-compilers-feedstock/blob/7ea10c2526445beb9faff23efcc6bc40fc0079b5/recipe/meta.yaml#L23=
-    - patches/remove_gcc_maximum_requirements.patch 
+    - patches/remove_gcc_maximum_requirements.patch
 
 build:
-  number: {{ number }}
+  number: {{ number + 100 }}  # [blas_impl == "mkl"]
+  number: {{ number }}  # [blas_impl != "mkl"]
   skip: true  # [win]
   skip: true  # [cuda_compiler_version == "10.2"]
   skip: true  # [cuda_compiler_version == "11.0"]
   skip: true  # [cuda_compiler_version == "11.1"]
+  skip: true  # [py != 310]
```

Comment:
move this to below line 37

Reply:
and copy to line 165

Reply:
This didn't appear to help.

Reply:
Adding a top-level requirements section with python did the trick. I wonder if this is a smithy bug or conda build bug?